test/integration-test/CodeGen/copy_stencil_benchmark.cpp

```diff
@@ -32,9 +32,12 @@ TEST(copy_stencil, test) {
   verif.fill(-1.0, out_gt, out_naive);
 
   gridtools::copy_stencil copy_gt(dom, in, out_gt);
+  verif.sync_storages(in);
   cxxnaive::copy_stencil copy_naive(dom, in, out_naive);
 
   copy_gt.run();
+
+  in.sync();
```

Comment:
could actually move down before the verification, right?

Reply:
I guess we discussed that already